src/search-lib/SearchDriver.js

```diff
@@ -172,7 +172,7 @@ function matchFilter(filter1, filter2) {
  * - facetConfig: Facet
  *   Configuration for Facet filters to be used within this application. The
  *   syntax for Facet configuration follows the API syntax:
- *   https://swiftype.com/documentation/app-search/api/search/facets. In
+ *   https://swiftype.com/documentation/search-lib/api/search/facets. In
```

Comment:
This was mistakenly edited during the replace all. :)

Reply:
Oh geez, good catch 🤦‍♂️